SDK/ViewControllers/CalendarItemViewController.swift

```diff
@@ -109,6 +110,17 @@ final class CalendarItemViewController: UIViewController {
         })
     }
     
+    //Scroll to the next upcoming Event
+    private func scrollToNextEvent(in calendar: ICalendar?) {
+        guard let nextUpcomingEvent = calendar?.events.filter({ $0.startDate > Date() }).first else { return }
```

Comment:
why the filter in the first place? Couldn't you just do it like this?
```
 let indexOfUpcomingEvent = calendar?.events.firstIndex { $0.startDate > Date() }
```
Also, I'd select it by the end date, so we don't skip ongoing events. I mean, if today is a holiday, we'd skip it if we go by the start. We need to ensure that all events actually have an end date. @rutgerg is that a reasonable assumption?

Reply:
Yes, all events in our ics have an end date. You might run into a problem when a long running event and a short running event overlap but I don't think we have that.

Reply:
This is not an overly important point now. Let's continue with notifications and fastlane for now and come back when we see it working